mitmproxy/command.py

```diff
@@ -86,6 +76,20 @@ def signature_help(self) -> str:
         return "%s %s%s" % (self.path, params, ret)
 
     def prepare_args(self, args: typing.Sequence[str]) -> typing.List[typing.Any]:
+
+        # Arguments that are just blank spaces aren't really arguments
+        # We need to get rid of those. If the user intended to pass a sequence
+        # of spaces, it would come between quotes
+        clean_args = []
+        for a in args:
+            if isinstance(a, str):
```

Comment:
`args` is defined as `typing.Sequence[str]` in the method signature, so this check seems superfluous?

Reply:
Even though it is defined like that, for some reason python is not checking it? One of the tests was passing the number 3 there, and the function accepted it (causing an error since a number can't be `split()`ed. I am not much of a python programmer, but seems to me that typing in python is just for annotating, not to enforce? Anyways, that is the reason why I added that check.

Reply:
Yes, Python's type system is an afterthought and not enforced. If we specify in the function signature that we expect a sequence of strings, we should just assume that this is true. Let's remove the test and the special case here.

Reply:
Alright. Done!